src/sys/signal.rs

```diff
@@ -490,11 +490,13 @@ pub enum SigevNotify {
 
 /// Used to request asynchronous notification of the completion of certain
 /// events, such as POSIX AIO and timers.
+#[cfg(not(target_os = "openbsd"))]
```

Comment:
Is this because libc doesn't support this on openbsd or is it because it's unsupported on OpenBSD?